<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>toolSetDialog</class>
 <widget class="QDialog" name="toolSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>17</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Microsoft Tai Le&quot;;</string>
   </property>
   <property name="text">
    <string>网站恶意代码监测工具</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton_1">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>40</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="manualButton_1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>使用手册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="manualButton_2">
   <property name="geometry">
    <rect>
     <x>376</x>
     <y>40</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>使用手册</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>17</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Microsoft Tai Le&quot;;</string>
   </property>
   <property name="text">
    <string>SQL注入验证检测工具</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="manualButton_3">
   <property name="geometry">
    <rect>
     <x>626</x>
     <y>40</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>使用手册</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>17</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Microsoft Tai Le&quot;;</string>
   </property>
   <property name="text">
    <string>系统漏洞检测工具</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton_3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>40</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="manualButton_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>使用手册</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>107</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Microsoft Tai Le&quot;;</string>
   </property>
   <property name="text">
    <string>网站安全检测工具</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton_4">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>130</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="manualButton_5">
   <property name="geometry">
    <rect>
     <x>376</x>
     <y>130</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>使用手册</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>107</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Microsoft Tai Le&quot;;</string>
   </property>
   <property name="text">
    <string>数据库安全检测工具</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton_5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="manualButton_6">
   <property name="geometry">
    <rect>
     <x>626</x>
     <y>130</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>使用手册</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>107</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Microsoft Tai Le&quot;;</string>
   </property>
   <property name="text">
    <string>主机病毒检测工具</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton_6">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>130</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="manualButton_7">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>220</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>使用手册</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>197</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Microsoft Tai Le&quot;;</string>
   </property>
   <property name="text">
    <string>主机木马检测工具</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton_7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="manualButton_8">
   <property name="geometry">
    <rect>
     <x>376</x>
     <y>220</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>使用手册</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>197</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Microsoft Tai Le&quot;;</string>
   </property>
   <property name="text">
    <string>抓包工具</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton_8">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="manualButton_9">
   <property name="geometry">
    <rect>
     <x>626</x>
     <y>220</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>使用手册</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>197</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Microsoft Tai Le&quot;;</string>
   </property>
   <property name="text">
    <string>日志分析工具</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton_9">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>220</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="manualButton_10">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>310</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>使用手册</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>287</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Microsoft Tai Le&quot;;</string>
   </property>
   <property name="text">
    <string>日志切割工具</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton_10">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="manualButton_11">
   <property name="geometry">
    <rect>
     <x>376</x>
     <y>310</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>使用手册</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>287</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Microsoft Tai Le&quot;;</string>
   </property>
   <property name="text">
    <string>文件恢复工具</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton_11">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>310</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>13</y>
     <width>56</width>
     <height>56</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/tool/image/tool/eydm.png</normaloff>:/tool/image/tool/eydm.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>56</width>
     <height>56</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>13</y>
     <width>56</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/tool/image/tool/sqlzr.png</normaloff>:/tool/image/tool/sqlzr.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>56</width>
     <height>56</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>13</y>
     <width>56</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/tool/image/tool/xtld.png</normaloff>:/tool/image/tool/xtld.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>56</width>
     <height>56</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>103</y>
     <width>56</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/tool/image/tool/wzaq.png</normaloff>:/tool/image/tool/wzaq.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>56</width>
     <height>56</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>103</y>
     <width>56</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/tool/image/tool/sjkaq.png</normaloff>:/tool/image/tool/sjkaq.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>56</width>
     <height>56</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>103</y>
     <width>56</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/tool/image/tool/zjbd.png</normaloff>:/tool/image/tool/zjbd.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>56</width>
     <height>56</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>193</y>
     <width>56</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/tool/image/tool/zjmm.png</normaloff>:/tool/image/tool/zjmm.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>56</width>
     <height>56</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>193</y>
     <width>56</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/tool/image/tool/zbgj.png</normaloff>:/tool/image/tool/zbgj.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>56</width>
     <height>56</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>193</y>
     <width>56</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/tool/image/tool/rzfx.png</normaloff>:/tool/image/tool/rzfx.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>56</width>
     <height>56</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>56</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/tool/image/tool/rzqg.png</normaloff>:/tool/image/tool/rzqg.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>56</width>
     <height>56</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>280</y>
     <width>56</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/tool/image/tool/wjhf.png</normaloff>:/tool/image/tool/wjhf.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>56</width>
     <height>56</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
